packages/access-api/src/models/spaces.js

```diff
@@ -21,7 +21,12 @@ export class Spaces {
   constructor(d1) {
     /** @type {GenericPlugin<SpaceRecord>} */
     const objectPlugin = new GenericPlugin({
-      metadata: (v) => JSON.parse(v),
+      metadata: (v) => {
+        if (v === 'EMPTY') {
```

Comment:
Why "EMPTY" and not just ""?

Reply:
https://github.com/web3-storage/w3protocol/issues/447

Reply:
I pushed a comment explaining the magic `EMPTY`